lib/core/src/server/babel-core-proxy.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/no-extraneous-dependencies,global-require */
+let babelCore = null;
+
+try {
+  babelCore = require('@babel/core');
```

Comment:
This can accidentally resolve to something only accessible from `node_modules/@storybook/core/node_modules` not from user's app.

We need to add `babel-core` peerDependency and ask users to install bridge if we need to use Babel directly

Reply:
That what I've tried to prevent, but I assume better to have a peer on a bridge. I will do this after the PR to `babel-merge`